Examples/Modules/laser_injection/analysis_laser.py

```diff
@@ -9,11 +9,15 @@
 
 
 import sys
+
 import matplotlib
+
 matplotlib.use('Agg')
```

Comment:
those lines seem to confuse it, but does not look like the end of the world to me

Reply:
https://github.com/PyCQA/isort/issues/295